config/plugins/centos.conf

```diff
@@ -1,3 +1,5 @@
 [CentOS]
 base-repo-url=http://vault.centos.org/centos/$releasever/os/Source/
 updates-repo-url=http://vault.centos.org/centos/$releasever/updates/Source/
+inactive-releases=5, 6
```

Comment:
With this the 5 and 6 are added to FAF (even if as EOL) and we start to process uReports from them. And we not really want that. _(If uReport comes from unknown OpSys, it is rejected and moved to `deferred` dir.)_


So I think the `incactive-releases` should be empty by default.

Reply:
removed

Reply:
removed

Reply:
removed